guardian/SDL/.gdnbaselines

```diff
@@ -0,0 +1,26 @@
+{
+  "version": "latest",
+  "baselines": {
+    "default": {
+      "name": "default",
+      "createdDate": "2022-09-09 14:14:48Z",
+      "lastUpdatedDate": "2022-09-09 14:14:48Z"
+    }
+  },
+  "results": {
```

Comment:
>results [](http://example.com/codeflow?start=3&length=7)

I am not familiar with Guardian, can you provide a quick explanation of this file? Also, this element is called "results" but it's not an array?

Reply:
Guardian is the team that owns the tasks based on some tools such as BinSkim, CredScan, PREFast, etc.

They work as a wrapped to expose the tools in AzureDevOps pipelines.

It is a KeyValuePair.
The key is a hash of the file, which allows you to deduplicate.

In this case, Payton created the suppression due to a false-positive in the tests that I also shared in the email.

Reply:
If the test can be updated and still work for you, then, yes, I agree that that should be done :)

Reply:
Changed and merged into main.